doc/getting-started.md

```diff
@@ -50,3 +68,21 @@ name:
 
     /join #soju/libera
     /join #somechannel/rizon
+
+### Detaching/Parting Channels
+
+You can detach from a channel by including detach in the /part message e.g. `/part #soju detach`
+
+When detached, Soju will keep your user joined to the channel and log activity, but your clients will no longer display the channel.
+
+To fully leave a channel issue a standard `/part #soju`
```

Comment:
Could you add a ref to `/msg BouncerServ help channel update` and the related man page section?